test/LibraryManager.Test/Providers/Cdnjs/CdnjsCatalogTest.cs

```diff
@@ -120,7 +120,7 @@ public void Setup()
             Assert.AreEqual(7, result.Start);
             Assert.AreEqual(0, result.Length);
             Assert.IsTrue(result.Completions.Count() >= 69);
-            Assert.AreEqual("1.2.3", result.Completions.Last().DisplayText);
+            Assert.AreEqual("jquery@1.2.3", result.Completions.Last().DisplayText);
```

Comment:
>jquery@1.2.3 [](start = 29, length = 12)

I don't know how this works. Is this information being obtained from a Mock?

Reply:
I think this is from _catalog.GetLibraryCompletionSetAsync.

---
In reply to: [198553292](https://github.com/aspnet/LibraryManager/pull/258#discussion_r198553292) [](ancestors = 198553292)

Reply:
Will change it back as to keep the cdnjs display behavior

---
In reply to: [198653075](https://github.com/aspnet/LibraryManager/pull/258#discussion_r198653075) [](ancestors = 198653075,198553292)